src/api/firebase/firebase.types.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable max-len */
+import firebase from 'api/firebase/firebase.api';
+
+export type Firestore = {
+  Doc: firebase.firestore.QueryDocumentSnapshot<firebase.firestore.DocumentData>;
+};
+
+// Interface for sample documents inside sample collection called "meals"
+export interface Meals {
```

Comment:
The interface should be named in the singular form (`Meal` instead of `Meals`).
Array or collection of objects could be named `meals`.

Reply:
Fixed.